extensions/prometheus-grafana-k8s/README.md

```diff
@@ -0,0 +1,98 @@
+# prometheus-grafana Extension
+
+
+This is the prometheus-grafana extension.  Add this extension to your json file as shown below to automatically enable prometheus and grafana in your new Kubernetes cluster.
+
+```
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_DS2_v2",
+      "extensions": [
+        { 
+          "name": "prometheus-grafana-k8s"
+        }
+      ]
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 3,
+        "vmSize": "Standard_DS2_v2",
+        "availabilityProfile": "AvailabilitySet"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "extensionProfiles": [
+      { 
+        "name": "prometheus-grafana-k8s", 
+        "version": "v1",
+        "rootURL": "https://raw.githubusercontent.com/ritazh/acs-engine/feat-monitor/"

```

Comment:
What's the purpose of the `rootURL` property? We probably want it to refer to something a little more persistent.

Reply:
That's where it'll pull the extension itself from. And absolutely that should be pointing to the upstream repo, but we were using @ritazh's fork to test this out.

This is a chicken and the egg thing. What is usually done to test out an extension in a fork before being merged upstream?

Reply:
Understood. Because this is an optional property, can we remove it from the examples/ api model, or is there value in configuring your cluster to include this extension with the `rootURL` property properly defined?

Reply:
`rootURL` is used by `dta = strings.Replace(dta, "EXTENSION_URL_REPLACE", extensionProfile.RootURL, -1)` in `engine.go` and `EXTENSION_URL_REPLACE` is used in `template-link.json` as:
```
"templateLink": {
            "uri": "EXTENSION_URL_REPLACEextensions/prometheus-grafana-k8s/v1/template.json",
            "contentVersion": "1.0.0.0"
        },
```